### ISSUE TITLE
I think lobby may be scraping zero-k's website for minimaps

### ISSUE THREAD

**Issue by MooseOnTheLoose** [beyond-all-reason/BYAR-Chobby#279]
![Screenshot_307](https://user-images.githubusercontent.com/1368663/196004446-da46a7c9-fd4c-4165-a01d-5935d31af593.png)

This is not the proper BAR minimap directory, why is it being populated? These are Zero-K minimaps, further investigation is required.

**Comment by MooseOnTheLoose** [comment]
```	if WG.WrapperLoopback and WG.WrapperLoopback.DownloadImage and (not VFS.FileExists(filePath)) then
		if not self.minimapDownloads[mapName] then
			Spring.CreateDir("LuaMenu/Images/Minimaps")
			WG.WrapperLoopback.DownloadImage({ImageUrl = "http://zero-k.info/Resources/" .. mapName .. ".minimap.jpg", TargetPath = filePath})
			self.minimapDownloads[mapName] = true
		end
		return filePath, true
	end```

From configuration.lua

**Comment by MooseOnTheLoose** [comment]
Fixed with #281
